.github/workflows/docker.yml

```diff
@@ -35,18 +35,16 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build main sliding sync image
-        if: github.ref_name == 'main'
-        id: docker_build_sliding_sync
-        uses: docker/build-push-action@v4
+      - name: Generate docker image tags
+        id: metadata
+        uses: docker/metadata-action@v5
         with:
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
```

Comment:
We seem to lose the caching..?

Reply:
I feel like this diff is just because i merged them into the one docker/build-push-action@v4

Though that being said, i think the `if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')` should be removed so its always pushing an email? or did you only want images pushed for releases? its been a few weeks.

Reply:
No wait yes, the if: should be removed so it always builds the image, but maybe the `push: true` could be made into a conditional so only pushes the result sometimes?